### PR TITLE
optimizations

### PR DESCRIPTION
**Pull request by Alexey-T**

### CODE REVIEW

**Comment by Alexey-T** [comment]
@andgineer Last optim makes benchmark runs 3% faster

File: src/RegExpr.pas [hunk 1]
```diff
@@ -3334,8 +3334,8 @@ function TRegExpr.ParseAtom(var flagp: integer): PRegExprChar;
           else
             Len := FindSkippedMetaLen(regparse + 1, fRegexEnd) + 1;
             // bad {n,m} - compile as EXACTLY
-        ender := (regparse + Len)^;
-        if (Len > 1) and ((ender = '*') or (ender = '+') or (ender = '?') or (ender = '{')) then
+        TempChar := (regparse + Len)^;
```
Comment (andgineer) [inline src/RegExpr.pas]: For me old name was self-documented and new one is not so clear..

Comment (andgineer) [reply src/RegExpr.pas]: > @andgineer Last optim makes benchmark run 3% faster

Great - this is scientific approach!

**Comment by Alexey-T** [comment]
@andgineer Fixed var name to nicer: EnderChar.

**Comment by Alexey-T** [comment]
@andgineer see #123 about last patch.

**Pull request merged**